tools/build-common.sh

```diff
@@ -128,6 +128,9 @@ get_target_build_options() {
         pl18ev)
             BUILD_OPTIONS+="-DPCB=PL18 -DPCBREV=PL18EV"
             ;;
+        nb4p)
+            BUILD_OPTIONS+="-DPCB=PL18 -DPCBREV=NB4P"
```

Comment:
Hmm... it is unintuitive to have to use PCB=PL18 for NB4+. Better would be to not have to specify PL18 here.

Reply:
Yes, this is next step, I planned to rename after #5397, since current #5397 can still use old nv14 target to build the firmware for testing and comparison.  When everything is stable, the rename will take place.